app/layout.tsx

```diff
@@ -19,7 +19,7 @@ export default function RootLayout({
     <html lang="en">
       <body className={lato.className}>
         <Providers>
-          <Box sx={{ position: "sticky", top: "0" }}>
+          <Box zIndex={10} sx={{ position: "sticky", top: "0" }}>
```

Comment:
@BrettEastman I believe the `top` property in the `sx` prop can be moved out of `sx` into a regular ChakraUI props. But am I missing something? 

I actually have never used `sx` before so I had to look it up -  It seems like the `sx` prop is meant for setting a CSS property that's not listed in the [Style Props](https://chakra-ui.com/docs/styled-system/style-props) list or for using a custom CSS variable. `top` is a ChakraUI prop with the same name as the CSS prop! 

I can refactor if you give me the green light, but wanted to double-check in case I missed something.

Reply:
@avinashi10 Go for it! Yes, actually it looks like both top and position are listed Style Props in Chakra, so the sx attribute isn't needed at all. Thanks for spotting that!